README.md

```diff
@@ -1008,7 +1027,7 @@ function getValue(MyObject $object)
 /**
  * @throws PaybackUnavailableException
  */
-function payback($requestId)
+function payback(int $requestId): bool
```

Comment:
in case we add the type hints and we use docBlock, shouldn't we fill in the full information there, not only the missing part? In case it's about wrong samples, then I am not sure why we also edit it :D

Reply:
we edit that wrong example because its wrong due to incorrect typehint.
so its wrong due to logical reasons, thats why all other conventions should be met